ubcpi/static/html/ubcpi_edit.html

```diff
@@ -0,0 +1,153 @@
+                            <input type="button" class="button action-primary add-image-button" value="Add Option" ng-click="esc.add_option()" name="pi-add-item-link" ng-hide="esc.data.options.length>=10"/>
+                        </label>
+                        <div class="ubcpi-options-list-container">
+                            <div class="wrapper-comp-setting ubcpi-options-list" id="option-{{ $index + 1}}" ng-repeat="option in esc.data.options track by $index">
+                                <label class="label setting-label pi-setting-label" for="pi-option-{{$index}}">Option {{$index + 1}}<span class="tip setting-help">Specify the value for this option. If you wish to add an image to this option click the Add Image button below.</span> <input type="button" class="button action-primary add-image-button" value="{{esc.data.options[$index].show_image_fields ? 'Remove Image' : 'Add Image to Option ' + ($index+1) }}" ng-click="esc.show_image_fields($index)" name="pi-option-{{$index}}" /></label>
+                                <input class="input setting-input pi-options" name="" id="pi-option-{{$index}}" ng-model="esc.data.options[$index].text" ng-model-options="{ debounce: 500 }" type="text" required />
+                                <a href="" class="remove-action remove-setting" aria-label="Remove Option" data-lang="af" data-value="" ng-click="esc.delete_option($index)" data-tooltip="Remove option"><i class="icon fa fa-times-circle"></i><span class="sr">Remove</span></a>
+
+                                <div class="add-image-inputs" ng-if="esc.data.options[$index].show_image_fields">
+                                	<label class="label setting-label pi-setting-label" for="question-text-image-url-{{$index}}">Image URL<span class="tip setting-help">Upload a file using Content > Files &amp; Uploads. Then copy the 'Studio URL'. Paste that into the text field above.</span></label>
+                                	<input type="text" name="question-text-image-url" id="question-text-image-url-{{$index}}" class="input setting-input pi-options image-input-url" ng-model="esc.data.options[$index].image_url" ng-model-options="{ debounce: 500 }" required/> <br />
+                                	<label class="label setting-label pi-setting-label" for="question-text-image-url-{{$index}}">Image Position<span class="tip setting-help">Choose if you want the image to appear above or below the question text.</span></label>
+                                	<select name="question-text-image-position" id="question-text-image-position-{{$index}}" ng-model="esc.data.options[$index].image_position" ng-options="position as desc for (position, desc) in esc.image_position_locations"></select><br />
+                                    <label class="label setting-label pi-setting-label" for="question-text-image-alt-{{$index}}">Image Description<span class="tip setting-help">Please provide alternative text for this image.</span></label>
```

Comment:
Nit: is it possible to get the helper text on the next line?

![screen shot 2015-09-16 at 2 25 43 pm](https://cloud.githubusercontent.com/assets/1985317/9914247/e447d5dc-5c7e-11e5-935f-f7d38b434e29.png)